components/stack/stack_constants.js

```diff
@@ -0,0 +1,18 @@
+export const DT_STACK_DIR = {
+  default: 'column',
+  column: 'column',
+  row: 'row',
+  'row-reverse': 'row-reverse',
+  'column-reverse': 'column-reverse',
+};
+
+export const DT_STACK_RESPONSIVE_BREAKPOINTS = ['sm', 'md', 'lg', 'xl'];
+
+export const DT_STACK_GAP = {
+  100: '100',
+  200: '200',
+  300: '300',
+  400: '400',
+  500: '500',
+  600: '600',
+};
```

Comment:
I avoid to use a method instead use a 'object directly access' due performance, I believe it would be better to use Set and just check if it exists with .has

I would prefer to avoid as much as we can to use those methods on components if its possible.

Anyways since the impact is insignificant I can update it and use .includes for the validation.